### DISCUSSION TITLE
Refresh/Reload CefSharp.Wpf

### DISCUSSION THREAD

**Discussion by Gianx99** [cefsharp/CefSharp#3743] [Q&A]
Good morning

how can I refresh (like F5 in real browser) CefSharp.Wpf browser via codebehind?
In a previuos version, there was a "Reload" method, but in the last version there is not.

Regards

**Comment by ukandrewc** [comment]
According to the Docs, it is still there in 91.1.xxx
http://cefsharp.github.io/api/91.1.x/html/T_CefSharp_Wpf_ChromiumWebBrowser.htm

**Comment by amaitland** [comment]
The Reload methods along with a number of others were converted to extension methods some years ago to reduce code duplication. 

Add a the following to your class and you'll be able to use the extension methods 

```
using CefSharp;
```

**Reply by Gianx99** [reply]
THANKS :-)
